databases.md

```diff
@@ -0,0 +1,32 @@
+# DATABASES
+
+
+## Prerequisite knowledge:
+NONE
+
+
+## Classification / Acceptance :
+
+#### Apprentice aka _Hello World_
+ * I know what a database is and what it is used for
+ * I can enumerate at least 5 advantages and disadvantages of using a database
+ * I know general requirements that each database must meet to work correctly
+ * I can enumerate at least 3 examples of SQL databases and 3 examples of NoSQL databases
```

Comment:
What about the understanding the differences between SQL and NoSQL? I think it's more important than just naming them.

Reply:
Ok. I added information to be able to tell major differences. I think however that the following line (in intermediate level) covers that point.
https://github.com/bethinkpl/bit-docs/pull/24/files#diff-063b65ec7c2204b9aadd81138ab83dc3R19